public/localscripts/parallel_histograms.js

```diff
@@ -0,0 +1,184 @@
+function parallel_histograms(selector, data, options) {
+    options = options || {};
+
+    _.defaults(options, {
+        width: 600,
```

Comment:
I did 600 x 370, which is the size of the "Duration statistics" histogram, and not the default for histograms, which is 100 x 40.